From d419a7e3c6c0b64fb1d95e5994b896c44c2d5e40 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:51:18 -0400
Subject: [pci] Update maintainer email address

Message-id: <20140806035118.1438.71944.stgit@gir.stowe>
Patchwork-id: 87476
O-Subject: [RHEL7.1 PATCH 67/69] PCI: Update my email address
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 05d8f6593d1d0b4677822136b61d48d7581c5d16

commit 05d8f6593d1d0b4677822136b61d48d7581c5d16
Author: Ben Hutchings <ben@decadent.org.uk>
Date:   Fri Apr 25 12:25:42 2014 -0600

    PCI: Update my email address

    Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Documentation/ABI/testing/sysfs-bus-pci b/Documentation/ABI/testing/sysfs-bus-pci
index a3c5a66..ab8d76d 100644
--- a/Documentation/ABI/testing/sysfs-bus-pci
+++ b/Documentation/ABI/testing/sysfs-bus-pci
@@ -117,7 +117,7 @@ Description:
 
 What:  /sys/bus/pci/devices/.../vpd
 Date:  February 2008
-Contact: Ben Hutchings <bhutchings@solarflare.com>
+Contact: Ben Hutchings <bwh@kernel.org>
 Description:
   A file named vpd in a device directory will be a
   binary file containing the Vital Product Data for the
-- 
1.7.1